src/libsync/propagateuploadng.cpp

```diff
@@ -400,6 +400,7 @@ void PropagateUploadFileNG::slotPutFinished()
         // the chunk sizes a bit.
         quint64 targetSize = (propagator()->_chunkSize + predictedGoodSize) / 2;
 
+        // Adjust chunk size for the item being classified to be chunked.
```

Comment:
this comment is wrong.  This has nothing to do with the classification, quite the contrary.  It is the actual size that will be used by the chunking-ng algorithm